alarm_backend/app.py

```diff
@@ -1,8 +1,11 @@
-from flask import Flask
+from flask import Flask, jsonify, request
 from models import db, connect_db, Alarm
+from helper import serialize
+from flask_cors import CORS
 
 
 app = Flask(__name__)
+CORS(app)
```

Comment:
Advance topic: Make sure you specify a list of origins here or anyone can access your backend https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS